tools/nni_trial_tool/trial_keeper.py

```diff
@@ -213,6 +215,22 @@ def run(self):
     fetch_file_thread.start()
 
 
+def _set_adaptdl_signal_handler():
+    import signal
+    global _trial_process
+    def _handler(signum, frame):
+        nni_log(LogType.Info, "RECEIVED SIGNAL {}".format(signum))
+        print(_trial_process.pid)
```

Comment:
use nni_log(LogType.Debug) if this is for debug purpose.

Reply:
done